web/app/perusopetus/OmatTIedotPerusopetuksenOppiaineet.jsx

```diff
@@ -88,15 +91,16 @@ const Oppiainetaulukko = ({model, suoritukset, showLaajuus}) => {
       </thead>
       <tbody>
       {
-        filteredSuoritukset.map(suoritus => (
+        Bacon.combineWith(isMobileAtom, mobile => filteredSuoritukset.map(suoritus => (
```

Comment:
Voisiko tätä selkeyttää antamalla `isMobileAtom`in suoraan `prop`sina? Tai edes käyttämällä `isMobileAtom`iin `map`ia? Nyt yhden muuttujan `combineWith` vaikuttaa monimutkaiselta.